### DISCUSSION TITLE
Install LML Pro using Forge - Permission denied

### DISCUSSION THREAD

**Discussion by petermaas** [spatie/laravel-medialibrary#3161] [Q&A]
I've added the  Composer Package Authentication details on my forge server but get the following error using Envoyer:

Using global Composer installation.
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 171 installs, 0 updates, 0 removals
  - Downloading spatie/laravel-medialibrary-pro (2.6.1)
 0/1 [>---------------------------]   0%    Failed to download spatie/laravel-medialibrary-pro from dist: The 'https://satis.spatie.be/dist/spatie/laravel-medialibrary-pro/spatie-laravel-medialibrary-pro-d99b87fad4ba06360a948391cfb1d978302c79f0-zip-241851.zip' URL required authentication (HTTP 401).
You must be using the interactive console to authenticate
    Now trying to download from source
  - Syncing spatie/laravel-medialibrary-pro (2.6.1) into cache

 1/1 [============================] 100%
In Git.php line 471:
                                                                               
  Failed to execute git clone --mirror -- 'git@github.com:spatie/laravel-medi  
  alibrary-pro.git' '/home/forge/.cache/composer/vcs/git-github.com-spatie-la  
  ravel-medialibrary-pro.git/'                                                 
                                                                               
  Cloning into bare repository '/home/forge/.cache/composer/vcs/git-github.co  
  m-spatie-laravel-medialibrary-pro.git'...                                    
  git@github.com: Permission denied (publickey).                               
  fatal: Could not read from remote repository.                                
                                                                               
  Please make sure you have the correct access rights                          
  and the repository exists.

**Comment by parallels999** [comment]
Try to download manually the package

**Comment by petermaas** [comment]
I've found the solution: Adding the public SSH key of the server to my github account.
